### PR TITLE
Migration guidance for `px()`

### LINKED ISSUE #5007: Create migration guidance for ThemeProvider

### LINKED ISSUE #5021: Create migration guidance for `px()` function

### PR DESCRIPTION
**Pull request by aveline**
### WHY are these changes introduced?

Fixes #5021 

### WHAT is this pull request doing?
Add migration guidance around the deprecation of `px()`

### CODE REVIEW

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 1]
```diff
@@ -190,6 +190,10 @@ We replaced a few of the following filter function instances with color tokens i
 | `ms-high-contrast-color('button-text-background')`   | `buttonFace`            |
 | `ms-high-contrast-color('background')`               | `window`                |
 
+#### `px()`
+
+All `px` values are now automatically converted to `rem` through our postcss config with the `postcss-pxtorem` plugin.
```
Comment (BPScott) [inline documentation/guides/migrating-from-v8-to-v9.md]: This is only true if the consumer has a postcss plugin that does this. This repo contains such a plugin but @shopify/postcss-plugin doesn't contain anything that does that, any app using sewing-kit would need to opt into using that plugin, which seems like a pretty disruptive migration suggestion.


Considering this is a migration guide for what to change in apps that consume polaris-react saying "the build in the  polaris-react repository" doesn't seem very relevant.

Comment (aveline) [reply documentation/guides/migrating-from-v8-to-v9.md]: 🤦‍♀️ of course. I'll update this to suggest copying the existing function. Unless you have a better suggestion?

Comment (BPScott) [reply documentation/guides/migrating-from-v8-to-v9.md]: Copying the existing function sounds like a good plan to me

**Comment by aveline** [comment]
Converting back to draft as this is now blocked by creation of the mega file.

**Review by lgriffee — APPROVED** [review]
🎉🚢✨💯

**Review by alex-page — APPROVED** [review]

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 2]
```diff
@@ -229,6 +229,14 @@ We replaced a few of the following filter function instances with color tokens i
 | `ms-high-contrast-color('button-text-background')`   | `buttonFace`            |
 | `ms-high-contrast-color('background')`               | `window`                |
 
+#### `px()`
+
+This function has been deprecated, but the definition can be copied and used locally.
+
+| Function | Source                                                                                                                                                           |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `px()`   | [definition](https://github.com/Shopify/polaris-react/blob/b443d114d447df15d9e72914c8ca5058439a175e/documentation/guides/legacy-polaris-v8-public-api.scss#L313) |
```
Comment (sam-b-rose) [inline documentation/guides/migrating-from-v8-to-v9.md]: Nice reference to the permalink! I think permalinks also support ranges, so we could highlight the entire `px()` function to make it more clear when landing on the page.

[#L313-L331](https://github.com/Shopify/polaris-react/blob/b443d114d447df15d9e72914c8ca5058439a175e/documentation/guides/legacy-polaris-v8-public-api.scss#L313-L331)

**Review by sam-b-rose — APPROVED** [review]

File: documentation/guides/migrating-from-v8-to-v9.md (hunk 2 above)
Comment (aveline) [reply documentation/guides/migrating-from-v8-to-v9.md]: Good suggestion! Thanks @samrose3

**Review by alex-page — APPROVED** [review]

**Review by aaronccasanova — APPROVED** [review]

**Pull request merged**